src/f4enix/input/d1suned.py

```
@@ -165,11 +168,114 @@ def get_irrad(self, daughter: str) -> Irradiation | None:
+
+        Raises
+        ------
+        ValueError
+            If scale_IRS is provided and more than one irradiation scenario is used.
+        """
+        # verify that either scale_IRS is None or only one irr scenario is provided
+        if scale_IRS is not None:
+            if len(irr_scenarios) > 1:
+                raise ValueError(
+                    "If scale_IRS is provided, only one irradiation scenario can be used."
+                )
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)